physionet-django/console/forms.py

```diff
@@ -760,6 +760,54 @@ def save(self):
         return section
 
 
+class StaticPageForm(forms.ModelForm):
+    """
+    Form for creating a dynamic static page."""
+
+    url = forms.RegexField(
+        label="URL",
+        max_length=100,
+        regex=r"^[-\w/\.~]+$",
+        help_text=(
```

Comment:
If possible, it would be good if the help text was also displayed on the "edit" page. Currently I only see it on the creation page.